app/src/org/commcare/android/framework/BreadcrumbBarFragment.java

```diff
@@ -360,6 +363,10 @@ public void onResume() {
             }
         }
     }
+
+    private boolean inLandscapeFormEntry(Activity a) {
+        return (a instanceof FormEntryActivity) && getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE;

```

Comment:
This function can be made static if you call `getResources()...` on the `a` argument